app/src/test/java/org/mozilla/fenix/components/metrics/AppStartupTypeTelemetryTest.kt

```diff
@@ -0,0 +1,118 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.components.metrics
+
+import io.mockk.MockKAnnotations
+import io.mockk.impl.annotations.RelaxedMockK
+import io.mockk.verify
+import org.junit.Before
+import org.junit.Test
+import org.mozilla.fenix.components.metrics.Event.AppStartupType.Type.HOT
+import org.mozilla.fenix.components.metrics.Event.AppStartupType.Type.WARM
+import org.mozilla.fenix.components.metrics.Event.AppStartupType.Type.COLD
+import org.mozilla.fenix.components.metrics.Event.AppStartupType.Type.COLD_SAVED_INSTANCE
+import org.mozilla.fenix.components.metrics.Event.AppStartupType.Type.WARM_SAVED_INSTANCE
+
+class AppStartupTypeTelemetryTest {
+
+    @RelaxedMockK
+    private lateinit var metricController: MetricController
+
+    private lateinit var appStartupTypeTelemetry: AppStartupTypeTelemetry
+
+    @Before
+    fun setup() {
+        MockKAnnotations.init(this)
+        appStartupTypeTelemetry = AppStartupTypeTelemetry(metricController)
+    }
+
+    @Test
+    fun `when FenixApplication and HomeActivity is created without savedInstance, then metric reports COLD startup type`() {
```

Comment:
nit: with `GIVEN WHEN THEN`, you should capitalize them to make it clear that you're following a convention, not just writing something coincidentally. The comma before then is also unnecessary with the convention.